pkg/kafka/interactive.go

```diff
@@ -14,16 +13,16 @@ const (
 	queryLimit = "1000"
 )
 
-func InteractiveSelect(ctx context.Context, connection connection.Connection, logger logging.Logger) (*kafkamgmtclient.KafkaRequest, error) {
+func InteractiveSelect(ctx context.Context, connection connection.Connection, logger logging.Logger, localizer localize.Localizer) (*kafkamgmtclient.KafkaRequest, error) {
 	api := connection.API()
 
 	response, _, err := api.Kafka().GetKafkas(ctx).Size(queryLimit).Execute()
 	if err != nil {
-		return nil, fmt.Errorf("unable to list Kafka instances: %w", err)
+		return nil, localizer.MustLocalizeError("kafka.common.error.couldNotFetchKafkas", localize.NewEntry("Error", err))
```

Comment:
We should try to avoid internationalizing the error like this. Returning the `error` object means it can be type checked and cast to a type. What's happening here is it is being converted to a string, meaning it can no longer be cast.

Reply:
Sure, done.